docs/qs_first_boot.md

```diff
@@ -0,0 +1,101 @@
+# OVOS Quick Start - First Boot
+
+You now have a flashed boot medium with OVOS installed.  Now what?
+
+Insert your boot medium, and power on your device.
+
+**NOTE** On a RPi4 with a USB device, it is recommended to use the top-right USB3 port.
```

Comment:
```suggestion
**NOTE** If using a Raspberry Pi 4 with a device conneected via USB-A, the top-right USB 3.2 Gen 1 port is recommended.
```

[Per the USB-IF, USB 3.0 is now USB 3.2 Gen 1.](https://www.diyphotography.net/usb-3-is-about-to-get-renamed-again-and-its-going-to-get-even-more-confusing/#:~:text=usb%203.0%2C%20which%20became%20usb%203.1%20gen%201%20is%20now%20usb%203.2%20gen%201) I know, we all hate it. We still need to use the official names though, surely?

By the way,

![This](https://assets.raspberrypi.com/static/blueprint-labelled-97975f4b1159239a8e248d180be87e3e.svg)

appears to demonstrate that the top-right port is actually the USB 2.0 port, whereas the middle port is the 3.2 Gen 1 port.